src/js/utils/styles.js

```diff
@@ -208,27 +208,72 @@ export const fillStyle = fillProp => {
   return undefined;
 };
 
-// focus also supports clickable elements inside svg
-export const focusStyle = css`
-  > circle,
-  > ellipse,
-  > line,
-  > path,
-  > polygon,
-  > polyline,
-  > rect {
-    outline: ${props =>
-        normalizeColor(props.theme.global.focus.border.color, props.theme)}
-      solid 2px;
-  }
-  outline-color: ${props =>
-    normalizeColor(props.theme.global.focus.border.color, props.theme)};
-  border-color: ${props =>
-    normalizeColor(props.theme.global.focus.border.color, props.theme)};
-  box-shadow: 0 0 2px 2px
-    ${props =>
-      normalizeColor(props.theme.global.focus.border.color, props.theme)};
-
+export const focusStyle = justBorder => css`
+  ${props =>
+    // focus also supports clickable elements inside svg
+    props.theme.global.focus &&
+    props.theme.global.focus.border &&
+    props.theme.global.focus.border.color &&
+    `> circle,
+    > ellipse,
+    > line,
+    > path,
+    > polygon,
+    > polyline,
+    > rect {
+      outline: ${normalizeColor(
+        props.theme.global.focus.border.color,
```

Comment:
Should use outline property from the theme.

Reply:
done